crates/api_models/src/payments.rs

```diff
@@ -86,10 +86,6 @@ pub struct PaymentsRequest {
     }))]
     pub routing: Option<serde_json::Value>,
 
-    /// This allows the merchant to manually select a connector with which the payment can go through
-    #[schema(value_type = Option<Vec<Connector>>, max_length = 255, example = json!(["stripe", "adyen"]))]
-    pub connector: Option<Vec<api_enums::Connector>>,
```

Comment:
@bernard-eugine, I think that it is safe to remove this field as it is not being used anywhere in the codebase, or should we let this be?